tests/lib/rules/no-restricted-html-elements.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * @author Doug Wade
+ */
+'use strict'
+
+const RuleTester = require('eslint').RuleTester
+const rule = require('../../../lib/rules/no-restricted-html-elements')
+
+const tester = new RuleTester({
+  parser: require.resolve('vue-eslint-parser'),
+  parserOptions: { ecmaVersion: 2015 }
+})
+
+tester.run('no-restricted-html-elements', rule, {
+  valid: [
+    {
+      filename: 'test.vue',
+      code: '',
+      options: ['error', 'button']
```

Comment:
Omit the first option, in the test. Because it always behaves as an 'error' in the test.
```suggestion
      options: ['button']
```